Cubical/Data/Prod/Properties.agda

```diff
@@ -20,6 +20,9 @@ private
 
 -- Swapping is an equivalence
 
+×≡ : {a b : A × B} → proj₁ a ≡ proj₁ b → proj₂ a ≡ proj₂ b → a ≡ b
+×≡ {a = (a1 , b1)} {b = (a2 , b2)} id1 id2 i = (id1 i) , (id2 i)
```

Comment:
Do you really need to match on a and b?

Reply:
No. I guess this is why it's better to use the product defined in Sigma.Base (for this we don't need to match). I don't use this product type anymore after Guillaume told me to use the Sigma version, so I guess this lemma is not really needed. But maybe it should be there anyway, just for completeness.

Reply:
Oh, I mean "yes", of course...

Reply:
Ah right, let's keep this one for completeness